docs/getting_started/scripts.md

```diff
@@ -36,12 +40,27 @@ said, keep them short, as they are displayed throughout the UI. Description
 supports Markdown formatting, and is where you should include examples on how to
 use your script.
 
+#### Common Scripts
+
+Common scripts are the basic building blocks for the flows.
+
 #### Trigger Scripts
 
-Scripts can be categorized as `Trigger` Scripts. It's a special category of
-Scripts, where the purpose is to pull data from an external source and compare
-it to the previously saved state. If there are changes, _trigger_ the rest of
-the flow, i.e. subsequent scripts.
+They are used as a first flow step most commonly with an internal state and a
+schedule to watch for changes on an external system, and compare it to the
+previously saved state. If there are changes, _trigger_ the rest of the flow,
+i.e. subsequent scripts.
+
+#### Approval Scripts
+
+Use them in order to suspend a flow until it's approved. Main usecase for
+approval scripts is to send an external notification with an URL that can be
+used to resume or cancel a flow. For more details check [Suspend/Resume a flow
+tutorial](../how-tos/6_suspend_resume_a_flow.md).
```

Comment:
I've left a placeholder here, let me know if I should remove it and add it later

Reply:
Looks good